app/javascript/src/Dashboard/ajax-widget.jsx

```diff
@@ -0,0 +1,34 @@
+// @flow
+
+import { widget } from 'Dashboard'
```

Comment:
I think that naming `widget` this method is wrong. The name suggest you'll get a widget but it only makes an ajax request.
I know you didn't added it in this PR, but we should change this in the future.
Just commenting